docs/content/latest/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -47,9 +47,57 @@ Use the `CREATE INDEX` statement to create an index on the specified columns of
 
 `CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.
 
+When an index is created on a populated table, YugabyteDB automatically
+backfills the existing data into the index.  In most cases, this uses an online
+schema migration.
+
+| | online | not online |
```

Comment:
```suggestion
| Condition | Online | Not online |
```

Reply:
Also, this table needs an introduction. What are you showing here?

"The following table explains..."